script/deploy/safe/deploy-safe.ts

```diff
@@ -311,22 +318,28 @@ const main = defineCommand({
     consola.info('Deployer:', walletAccount.address)
 
     // Determine EVM version
-    const networkConfig = networks[networkName.toLowerCase()]
+    const networkConfig = Object.prototype.hasOwnProperty.call(
```

Comment:
maybe create a small helper function for this as this since this is being used very often?